js/front/dataset/index.js

```diff
@@ -28,6 +28,9 @@ function parseUrl(url) {
     return a;
 }
 
+const RESOURCE_REGEX = /^#resource-([0-9a-f-]{36})$/;
+const RESOURCE_COMMUNITY_REGEX = /^#resource-community-([0-9a-f-]{36})$/;
```

Comment:
I think both regexp might be combined into a single one:
 ```javascript
const RESOURCE_REGEX = /^#resource(?:-community)?-([0-9a-f-]{36})$/
```

Reply:
`const RESOURCE_REGEX = /^#resource(-community)?-([0-9a-f-]{36})$/` will permit to extract in a single pass the ID and the fact that's it's a community resource or not.